feature-libs/organization/administration/components/unit/form/unit-form.component.ts

```diff
@@ -78,7 +83,9 @@ export class UnitFormComponent implements OnInit {
   constructor(
     protected itemService: ItemService<B2BUnit>,
     protected unitService: OrgUnitService
-  ) {}
+  ) {
+    useFeatureStyles('a11yApprovalProcessWithNoClearable');
```

Comment:
We shouldn't need it, if there are no style changes

Reply:
We should keep it because there is a style change.